static/app/components/onboardingWizard/taskConfig.tsx

```diff
@@ -291,6 +293,21 @@ export function getOnboardingTasks({
       actionType: 'app',
       location: `/organizations/${organization.slug}/projects/new/`,
       display: true,
+      SupplementComponent: withApi(({task}: FirstEventWaiterProps) => {
```

Comment:
Can you use the `useApi` hook instead?

Reply:
I could refactor this in a follow-up. For now I did not want to change it but reuse what we have in the code